Dockerfile

```diff
@@ -1,7 +1,58 @@
-# Inspired by https://container-solutions.com/faster-builds-in-docker-with-go-1-11/
-# Base build image
-FROM golang:1.14.4-alpine3.12 AS build_base
-RUN apk add bash make git curl unzip rsync libc6-compat gcc musl-dev lsof
+FROM ubuntu:18.04 AS linux
+ENV DEBIAN_FRONTEND noninteractive
+ENV SHELL /bin/bash
+ARG TZ=US/Eastern
+ENV TZ $TZ
+USER root
+RUN bash -c "for i in {1..9}; do mkdir -p /usr/share/man/man\$i; done" \
+ && echo 'APT::Get::Assume-Yes "true";' > /etc/apt/apt.conf.d/90noninteractive \
+ && echo 'DPkg::Options "--force-confnew";' >> /etc/apt/apt.conf.d/90noninteractive \
+ && apt-get update --fix-missing \
+ && apt-get install -qy --no-install-recommends \
+    ca-certificates \
+    tzdata \
+    locales \
+    git \
+    bash \
+    sudo \
+    unzip \
+    make \
+    curl \
+    procps \
+    net-tools \
+    apt-transport-https \
+    # -- it allows to start with nvidia-docker runtime --
+    libnvidia-compute-390 \
```

Comment:
I assume we need all of these deps on the machine where we build the binary, but we surely don't need all of this to RUN the binary!

If we do - will home users need to also install all of this to run Spacemesh?

If we don't - let's install this on the `golang` image instead of `linux` to make the shipped artifact leaner.

Reply:
Yeh. I used it for trubleshutting